cleanlab/object_detection/rank.py

```diff
@@ -464,8 +489,8 @@ def _get_valid_inputs_for_compute_scores_per_image(
 
 def _get_valid_inputs_for_compute_scores(
     alpha: float,
-    labels: Optional[List[Dict[str, Any]]] = None,
-    predictions: Optional[List[np.ndarray]] = None,
+    labels: Optional[List[Dict[str, Any]]],
```

Comment:
Why are labels and predictions no longer set to None if the param is not passed in?

Reply:
new mypy throws an error, (even though you're checking for None later in this function)

Reply:
I'll try resolving this